### ISSUE TITLE
Coal balls

### ISSUE THREAD

**Issue by ekrimmel** [paleo-data/happy-hours#16]
## What is your idea for a happy hour topic?
Tie into what is happening with the Coal Ball Working Group? Came up from discussion in the Pteridophyte TCN in 2023.

## Who could be involved in leading this happy hour?
@ljwalker, Margaret Landis

## Are there any timing considerations?
_[add text here - e.g. needs to happen before xyz conference, after field season, ASAP, etc.]_

## Next steps
This issue captures a happy hour topic idea. Once you are ready to finalize an idea and put it on the PDWG happy hour schedule, please create a new issue using the [Plan a happy hour](https://github.com/paleo-data/happy-hours/issues/new?assignees=&labels=PDWG&projects=&template=plan-happy-hour.md&title=%5Breplace+this+text+with+the+happy+hour+topic%5D) template. **Then, close this issue.**
